src/main/kotlin/sliv/tool/common/Extensions.kt

```diff
@@ -0,0 +1,16 @@
+package sliv.tool.common
+
+import javafx.scene.paint.Color
+
+fun Long.toColor() : Color {
+    val rgbRange = Pair(0, 255)
+    val red = (this % 2).toIntFromRange(rgbRange)
+    val green = (this % 3).toIntFromRange(rgbRange)
+    val blue = (this % 2).toIntFromRange(rgbRange)
+    return Color.rgb(red, green, blue)
```

Comment:
why do you store color as long?

Reply:
I have long uid and i intend to get three numbers from 0 to 255 to produce rgb color.

Reply:
why they are stored inside uid?